fcrepo/fcrepo
# add rebuild validation

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3560

**This PR will not build without https://github.com/fcrepo/fcrepo-storage-ocfl/pull/28**

# What does this Pull Request do?

Adds a validation step to rebuild/reindex. OCFL objects that are not valid F6 objects will not be accepted.

# How should this be tested?

Create an invalid F6 and either attempt to start F6 on top of it or attempt to side load it. The object should not be indexed and there should be errors in the logs indicating why the object was not accepted.

# Interested parties
@fcrepo/committers
